src/layers/fp64/scatterplot-layer/scatterplot-layer.js

```diff
@@ -65,20 +65,6 @@ export default class ScatterplotLayer extends Layer {
       layerHeight: {size: 2, update: this.calculateLayerHeight},
       instanceColors: {size: 3, update: this.calculateInstanceColors}
     });
-
-    // NVIDIA workaround
-    this.setUniforms({
-      ONE: 1.0
-    });
-  }
-
-  didMount() {
-    this.updateUniforms();
-  }
-
-  willReceiveProps(oldProps, newProps) {
-    super.willReceiveProps(oldProps, newProps);
-    this.updateUniforms();
```

Comment:
Yep. more precisely, the new mechanism is that is being returned by viewport.getUniforms() which is what is passed in to draw()